templates/deployment.yaml

```diff
@@ -673,11 +673,11 @@ spec:
            do
             filename=$(echo $FILE | cut -d '/' -f3 | cut -d '_' -f1)
             timestamp=$(echo $FILE | cut -d '_' -f3 | cut -d '.' -f1 |  sed 's/./&-/4;s/./&-/7;s/./&_/10;s/./&h/13;s/./&m/16;s/./&s/19')
-            date=$(echo $timestamp | cut -d '_' -f1)
+            {{ if .Values.rclone.dateInPath }}date=$(echo $timestamp | cut -d '_' -f1){{ else }}{{ end }}
             format=$(echo $FILE | rev | cut -d '.' -f1 | rev)
-            echo "$FILE is finished. Moving to data/finished/. Attempting to push to $RCLONE_REMOTE_NAME:$RCLONE_REMOTE_PATH/{{ .Values.labels.name }}/${date}/${timestamp}_{{ .Values.labels.name }}_{{ .Values.labels.cluster }}_${filename}.${format}"
+            echo "$FILE is finished. Moving to data/finished/. Attempting to push to $RCLONE_REMOTE_NAME:$RCLONE_REMOTE_PATH/{{ .Values.labels.name }}/{{ if .Values.rclone.dateInPath }}${date}{{ else }}{{ end }}/${timestamp}_{{ .Values.labels.name }}_{{ .Values.labels.cluster }}_${filename}.${format}"
```

Comment:
what is the `{{ else }}{{ end }}` good for?

Reply:
Good catch, removed.